cms/standard_pages/models.py

```diff
@@ -0,0 +1,39 @@
+from typing import ClassVar
+
+from django.db import models
+from wagtail.admin.panels import FieldPanel
+from wagtail.search import index
+
+from cms.core.blocks.stream_blocks import CoreStoryBlock
+from cms.core.fields import StreamField
+from cms.core.models import BasePage
+
+
+class InformationPage(BasePage):  # type: ignore[django-manager-missing]
+    """A generic information page model."""
+
+    template = "templates/pages/information_page.html"
+
+    summary = models.TextField(max_length=255)
+    last_updated = models.DateField(blank=True, null=True)
+    content = StreamField(CoreStoryBlock())
+
+    content_panels: ClassVar[list[FieldPanel]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        FieldPanel("last_updated"),
+        FieldPanel("content"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("summary"),
+        index.SearchField("content"),
+    ]
+
+    parent_page_types: ClassVar[list[str]] = [
```

Comment:
suggestion: move this at the top of the class, just after `template`. This helps with a quick scan

Reply:
done 🫡